README.md

```diff
@@ -155,7 +155,7 @@ cookbooks/travis_ci_connie
 ├── recipes
 │   └── default.rb
 └── spec
-    ├── ...
+	├── ...
```

Comment:
Unintentional indentation? 😁

Reply:
That got caught up in my automatic whitespace cleanup, but it didn't look off, so i left it. I can take it out though - just say the word :)

Reply:
On the other hand, it does look a little off, I'll fix it, sorry about that.